engines/python/setup/djl_python/transformers_neuronx.py

```diff
@@ -130,6 +128,11 @@ def parse_input(self, inputs):
                         raise ValueError(
                             "In order to enable dynamic batching, all input batches must have the same parameters"
                         )
+
+                if not "output_formatter" in param:
+                    param[
+                        "output_formatter"] = self.properties.output_formatter
```

Comment:
Hi @xyang16, self.properties is not defined for TransformersNeuronXService here. Apologies if this is already planned and I am missing context.